Filed/Behaviorism.md

```diff
@@ -0,0 +1,5 @@
+**_Definition_** :: Theory of learning that stresses the importance of having a particular form of behavior reinforced by someone other than the learner to shape or control what is learned.
+**_Source_** :: [AIH Ch3](app://obsidian.md/AIH%20Ch3)
```

Comment:
I'm used to seeing links in the form `[[AIH Ch3]]`, not a app:// link. Not sure if it matters, but maybe prefer the [[]] style if it works as well?